### PR TITLE
Allow for subsets of parameters to be fit

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
I don't have a way to test this because there's no example schema for which this is the case, but in a real example there was some distributions not set, so we are assuming those parameters will not be fit.

File: src/operations.jl [hunk 1]
```diff
@@ -52,7 +52,10 @@ function amr_get(amr::JSON3.Object, ::Type{ODESystem})
         push!(eqs, ofunc ~ expr)
     end
 
-    structural_simplify(ODESystem(eqs, t, allfuncs, paramvars; defaults = [statefuncs .=> initial_vals; sym_defs], name=Symbol(amr.name)))
+    sys = structural_simplify(ODESystem(eqs, t, allfuncs, paramvars; defaults = [statefuncs .=> initial_vals; sym_defs], name=Symbol(amr.name)))
+    @info sys
```
Comment (joshday) [inline src/operations.jl:56]: I've been trying to include at least the function name that the `@info` is contained in for ease of lookup.

**Review by joshday — APPROVED** [review]

File: src/operations.jl (hunk 1 above)
Comment (joshday) [reply src/operations.jl:56]:
```suggestion
    @info "amr_get(amr, ODESystem) --> $sys"
```

Comment (ChrisRackauckas) [inline src/operations.jl:56]:
```suggestion
    sys = structural_simplify(ODESystem(eqs, t, allfuncs, paramvars; defaults = [statefuncs .=> initial_vals; sym_defs], name=Symbol(amr.name)))
    @info "The AMR parsed the ODESystem to: $sys"
```

**Pull request merged**